app/views/images/index.html.erb

```diff
@@ -1,10 +1,12 @@
-<div class = "album text-muted">
-  <div class = "container">
-    <div class="row">
+<div class="album">
+  <div class="container-fluid">
+    <div class="row align-items-center">
       <% @images.each do |image| %>
-        <div class="card">
-          <%= image_tag image.imageurl, class: "mx-auto my-auto" %>
-        </div>
+        <a href=<%= "#{images_path}/#{image.id}" %>>
```

Comment:
The major change here was to turn each card into a link that navigates to the image's "show" page.